configure.ac

```diff
@@ -6,6 +6,7 @@ AC_CONFIG_HEADER(config.h)
 AM_INIT_AUTOMAKE([1.11 no-dist-gzip dist-xz foreign])
 AM_SILENT_RULES(yes)
 LT_INIT
+AC_PROG_CC_STDC
```

Comment:
What is occurring with this not set?

Reply:
Without that being set building lightdm v1.25.1 fails to build on RHEL7 / CentOS7, because the changes in code require the compiler to be run in `-std=c99` mode.